easybuild/easyconfigs/s/SPAM/SPAM-0.8.1.4_spambind_typo_fix.patch

```diff
@@ -0,0 +1,22 @@
+Fixes a copy-paste typo of spambind
```

Comment:
@PetrKralCZ Let's ask upstream why this wasn't a problem for them?
https://gitlab.com/spam-project/spambind/

Reply:
It's a bit difficult to tell, but I'm not sure whether the proposed patch is correct, let's see if we can get some input from upstream on this via https://gitlab.com/spam-project/spambind/-/issues/2 ...